src/parser/paladin/retribution/modules/core/WakeofAshes.js

```diff
@@ -23,10 +23,6 @@ class WakeofAshes extends Analyzer {
 
     constructor(...args) {
         super(...args);
-        this.active = this.selectedCombatant.hasTalent(SPELLS.WAKE_OF_ASHES.id);
-        if (!this.active) {
-            return;
-        }
         this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.WAKE_OF_ASHES), this.onWakeofAshesDamage);
```

Comment:
I can't comment directly on it 
but you should probably change the STATISTIC_ORDER to CORE from UNIMPORTANT()